src/e3/anod/checkout.py

```diff
@@ -20,6 +21,8 @@
     from collections.abc import Callable
     from e3.mypy import assert_never
 
+    SupportedVCS = Literal["git"] | Literal["svn"] | Literal["external"] | Literal["s3"]
```

Comment:
Would an `Enum` do it ? Use of `Literal` may be hard to cope with in some cases (type issue).

Reply:
That would be a backward incompatible change. Not sure this is worth it for just type-checking improvements.

Reply:
Agree here with olivier.